### ISSUE TITLE
question about file ‘trainer.py’

### ISSUE THREAD

**Issue by ShamLich** [HongwenZhang/PyMAF#38]
in line122
self.valid_loader = DataLoader(
            dataset=self.valid_ds,
            batch_size=cfg.TEST.BATCH_SIZE,
            shuffle=False,
            num_workers=cfg.TRAIN.NUM_WORKERS,
            pin_memory=cfg.TRAIN.PIN_MEMORY,
            sampler=val_sampler
        )
the batch_size of valdata is cfg.TEST.BATCH_SIZE

so the code in line 537
pbar = tqdm(desc='Eval', total=len(self.valid_ds) // cfg.TRAIN.BATCH_SIZE)

should be

pbar = tqdm(desc='Eval', total=len(self.valid_ds) // cfg.TEST.BATCH_SIZE)

right？

**Comment by HongwenZhang** [comment]
Hi, you are right. This bug affects the display of the progress bar. Thank you for pointing this out.
